src/clients/inst_autoinit.rb

```diff
@@ -196,6 +197,22 @@ def processProfile
 
       Builtins.y2debug("Autoinstall control file %1", Profile.current)
 
+      unsupported_sections = Y2ModuleConfig.unsupported_profile_sections
```

Comment:
Moving this code to a method would be nice (to not add more lines to the `main` method :) ).